README.md

```diff
@@ -1,2 +1,74 @@
 # K8ssandra Operator
-This is the Kubernetes operator for K8ssandra.
\ No newline at end of file
+This is the Kubernetes operator for K8ssandra.
+
+K8ssandra is a Kubernetes-based distribution of Apache Cassandra that includes several tools and components that automate and simplify configuring, managing, and operating a Cassandra cluster.
+
+K8ssandra includes the following components:
+
+* [Cassandra](https://cassandra.apache.org/)
+* [Stargate](https://stargate.io/)
+* [Medusa](https://github.com/thelastpickle/cassandra-medusa)
+* [Reaper](http://cassandra-reaper.io/)
+* [Grafana](https://grafana.com/)
+* [Prometheus](https://prometheus.io/)
+
+K8ssandra 1.x is configured, packaged, and deployed via Helm charts. Those Helm charts can be found in the [k8ssandra](https://github.com/k8ssandra/k8ssandra) repo.
+
+K8ssandra 2.x will be based on the this operator.
+
+# Contributing
+For anything specific to K8ssandra 1.x, please create the issue in the [k8ssandra](https://github.com/k8ssandra/k8ssandra) repo. 
+
+For more info on getting involved with K8ssandra, please check out the [k8ssandra community](https://k8ssandra.io/community/) page.
+
+The remainder of this section focuses on development of the operator itself.
+
+## Requirements
+* Go >= 1.16
+* kubectl >= 1.17
+* kustomize >= 4.0.5 
+* kind >= 0.11.1
+* Docker
+
+## Type definitions
+The Go type definition for custom resources live under the `api` directory in files with a `_types.go` suffix. The CRDs are derived from the structs defined in these files.
+
+## Updating CRDs
+As mentioned previously, the CRDs are generated based off the contents of the `_types.go` files. The CRDs live under `config/crd`.
+
+Run `make manifest` update CRDs.
+
+**Note:** Any changes to the `_types.go` files should be followed by `make manifests`
```

Comment:
```suggestion
**Note:** Any changes to the `_types.go` files should be followed by `make manifests`.
```

Reply:
Shouldn't we also do `make generate` when we change the API types?

Reply:
good catch